src/App.vue

```diff
@@ -84,6 +84,14 @@
       :webChatUrl="window['webChatUrl']"
     />
 
+    <!-- Display the Genesys WebMessage for SP/GP registrations only -->
+    <GenesysWebMessage
+      v-if="getFilingType === FilingTypes.REGISTRATION"
+      genesysURL="https://apps.cac1.pure.cloud/genesys-bootstrap/genesys.min.js"
+      environmentKey="cac1"
+      deploymentKey="42ed05f4-d545-436b-ba2c-94b66ed3396f"
+    />
```

Comment:
Don't hard-code this -- get it from the configuration.json file.

Reply:
Since this is a public forum, should we delete and create a new deployment key out of an abundance of caution?

Reply:
My fault, I forgot about this! Should be okay now.

Reply:
> Since this is a public forum, should we delete and create a new deployment key out of an abundance of caution?

This is a dev key -- I don't care if someone else sees it. If we do get spammed then we can change it. (I also don't want to destabilize Maximus by asking them for a new key.)

Reply:
Hey Karim, I know this is an easy mistake to make.  I've come very close to committing secrets to public repos in Github before. The challenge is that Github history is difficult to delete.  When we squash and merge this PR into the main branch, there will be no trace of this key in the commit history, but it will remain in your personal Github branch and (I'm guessing) this PR too.  When I worked on a previous BCGov project, they took the disclosure of secrets -- even in DEV -- seriously and it had to be reported even when no personal information was disclosed.  If it's our decision whether or not to report it, then I'm happy to forget about it.  As Severin says the downside risk is small.

Reply:
I don't know who we'd report this to. (And yes, I posted the keys first.)

Can we just forget about it? And if the dev webchat endpoint gets spammed, then we reset the keys?

Reply:
Thinking about this further, the deployment key can't be a secret -- it's available to anyone who knows where to look in JavaScript.  This definitely doesn't need to be reported.  Sorry if I caused any alarm.

Reply:
No, good question. Keep asking.

The key won't be in the JS but it will be in the configuration.json that is fetched.

Reply:
@jonathan-longe That's actually good to know. Thanks for bringing that up Jonathan.

Reply:
I'm satisfied in this conversation -- resolving it.

Jonathan, please open a new conversation if you want to continue chatting about this.